### DISCUSSION TITLE
Error pages 404/500/...

### DISCUSSION THREAD

**Discussion by cavasinf** [kevinpapst/TablerBundle#44] [Ideas]
I've added default error page from Tabler: https://preview.tabler.io/error-404.html
Was surprised they weren't in TablerBundle.

Are there any plans to have them later?
I can PR mine.

The problem is all the translations 😨. I'm not into that.

**Comment by kevinpapst** [comment]
You were suprised? I did not even think about them until now that I read your post 😁 

Can you prepare the translation for English and French? I'll do german and the rest needs to be done by the community.

**Reply by kevinpapst** [reply]
> I can PR mine.

Yes please 👍
